protos/flyteidl/admin/node_execution.proto

```diff
@@ -211,6 +215,10 @@ message NodeExecutionGetDataResponse {
     // Full_outputs will only be populated if they are under a configured size threshold. 
     core.LiteralMap full_outputs               = 4;
 
+    // String location uniquely identifying where the deck HTML file is.
+    // NativeUrl specifies the url in the format of the configured storage provider (e.g. s3://my-bucket/randomstring/suffix.tar)
+    string deck_uri                            = 5;
```

Comment:
Why do we need the URI in both Closure and GetDataResponse? isn't it only needed in one?

Reply:
`deck_uri` is saved in the Node Execution table and in the closure column, so we have to add it in closure to unmarshal it. https://github.com/pingsutw/flyteadmin/blob/a3f3e4e4e626233f3e79596679cf047d0a1e13a9/pkg/repositories/transformers/node_execution.go#L271

Reply:
aha... ic... then I think we shouldn't have it here... we should just return it in the node execution closure... this is not "data" anyway...

Reply:
ic, removed it.